test/timeout.bats

```diff
@@ -26,16 +26,16 @@ EOF
 }
 
 @test "should not clean up pod after timeout" {
-	create_conmon 2s
+	create_conmon 10s
```

Comment:
changing this from 2 to 10 will not fix the cause of the flake. The flake happens because the  rest of the operation takes a variable amount of time. 

Basically, if you look at the failures, crio is stuck on the conmon call. that's because setting up the network and container rootfs took > 8 seconds. Then, conmon was still in the middle of sleeping when we got past the `sleep 10s` below, and then attempted to call `runc list`, which returned nothing (because conmon hadn't finished creating the container)

The best course of action is to bump the sleep below to 20s, but leave these as 2s. I had been trying to minimize the sleeps below to get the tests running faster, but I suppose there's no way to expect it'd take that little time

Reply:
So, the reason why I increased the 2s timeouts is not because of conmon, but because of `crictl create -T`. I have seen that 1 second timeout is too small for our CI, and thought that maybe 2s might also too small sometimes, and increased it just in case. But once we increase `crictl create` timeout, conmon sleep needs to be increased.

Reverting 2s -> 10s change, let's see.

Reply:
the `context timeout exceeded` error we see at the top of the output is actually the correct behavior btw. we want the create_conmon timeout to equal the -T given to crictl, so we are guaranteed to timeout.

Reply:
Yes, I understand that. Although I have seen `context timeout exceeded` error from `crictl create -T 1s` when it merely tries to connect to cri-o, and increased the timeout to avoid that one.

Reply:
Commit updated.